background/ip-addresses.rst

```diff
@@ -0,0 +1,69 @@
+.. _infrastructure-ip-addresses:
+
+IP adresses
+===============================
+
+Divio's containerised infrastructure is distributed over a number of geographical and vendor based regions. At any one
+time, the servers within each region are allocated an IP address from that region's range.
+
+Servers in our infrastucture (application builders and runners, and other services) are not permanent hosts but
+short-lived instances that can be provisioned at a moment's notice and are regularly recycled.
+
+The numbers in the pool of IP addresses within a region are therefore constantly being reallocated to different
+servers, and there is no guarantee that a server's address will remain the same from one moment to another.
+
+In addition, vendors' IP ranges themselves are not guaranteed and can be subject to change regularly and frequently,
+sometimes without notice.
+
+This means that **IP addresses are not to be relied upon as a means of reaching or identifying Divio servers**.
+
```

Comment:
In general, when a server gets an IP, it keeps it until we deprovision the server itself. However, an app can be moved from a server to another at any moment. Not sure if you want to mention this or threat it as an implementation detail.